sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -116,20 +116,15 @@ export class ApplicationApi extends HttpBaseClient {
     applicationId: number,
     payload: SaveStudentApplicationDto,
   ): Promise<void> {
-    // this.apiClient.patch is used to catch the errors
-    // this errors are displayed in client side in toast message
-    await this.apiClient
-      .patch(
+    try {
+      await this.apiClient.patch(
```

Comment:
you can use `patchCall` and this function will catch and throw the error. same in `sources/packages/web/src/services/http/ProgramInfoRequestApi.ts` line 28
![image](https://user-images.githubusercontent.com/77353155/163248137-a4b801b0-2685-494a-9d25-437f3cd3a3e0.png)